get.js

```diff
@@ -1,7 +1,42 @@
 var _ = require('slapdash')
-var http = require('@qubit/http-api')
+var qubitApi = require('@qubit/qubit-api')
 var getLocale = require('./getLocale')
 
+var query = [
```

Comment:
Can use es6, so use something like
```js
const query = `
  query () {
    stuff ()
  }
`
```

Reply:
Nice didn't know we supported that in packages - guess the move to webpack sorts that out. Shall update